src/more_collectors.rs

```diff
@@ -84,7 +84,7 @@ impl Collector for StatsCollector {
     }
 
     fn requires_scoring(&self) -> bool {
-        true
+        false
```

Comment:
Don't know why this was still here. We don't need scoring in the stat collector.  @Sidhant29 could you confirm that my thinking is correct here, or have I forgotten something?

Reply:
@Sidhant29